>>> 'Pkg.add("CauseMap")' log
INFO: Cloning cache of CauseMap from git://github.com/cyrusmaher/CauseMap.jl.git
INFO: Installing BinDeps v0.4.7
INFO: Installing CauseMap v0.0.3
INFO: Installing Conda v0.5.3
INFO: Installing MacroTools v0.3.7
INFO: Installing PyCall v1.11.1
INFO: Installing SHA v0.3.3
INFO: Installing URIParser v0.1.8
INFO: Building Conda
INFO: Building PyCall
INFO: Using the Python distribution in the Conda package by default.
To use a different Python version, set ENV["PYTHON"]="pythoncommand" and re-run Pkg.build("PyCall").
Fetching package metadata .........
Solving package specifications: .

# All requested packages already installed.
# packages in environment at /home/vagrant/.julia/v0.4/Conda/deps/usr:
#
numpy                     1.13.0                   py27_0  
INFO: PyCall is using /home/vagrant/.julia/v0.4/Conda/deps/usr/bin/python (Python 2.7.13) at /home/vagrant/.julia/v0.4/Conda/deps/usr/bin/python, libpython = /home/vagrant/.julia/v0.4/Conda/deps/usr/lib/libpython2.7
INFO: /home/vagrant/.julia/v0.4/PyCall/deps/deps.jl has not changed
INFO: /home/vagrant/.julia/v0.4/PyCall/deps/PYTHON has not changed
INFO: Package database updated
INFO: METADATA is out-of-date â€” you may not have the latest version of CauseMap
INFO: Use `Pkg.update()` to get the latest versions of your packages

>>> 'Pkg.test("CauseMap")' log
Julia Version 0.4.7
Commit ae26b25 (2016-09-18 16:17 UTC)
Platform Info:
  System: Linux (x86_64-pc-linux-gnu)
  CPU: Intel(R) Xeon(R) CPU E3-1241 v3 @ 3.50GHz
  WORD_SIZE: 64
           Ubuntu 14.04.5 LTS
  uname: Linux 3.13.0-119-generic #166-Ubuntu SMP Wed May 3 12:18:55 UTC 2017 x86_64 x86_64
Memory: 2.9392738342285156 GB (968.6484375 MB free)
Uptime: 12142.0 sec
Load Avg:  0.91748046875  0.95849609375  0.9189453125
Intel(R) Xeon(R) CPU E3-1241 v3 @ 3.50GHz: 
       speed         user         nice          sys         idle          irq
#1  3500 MHz     514839 s       3283 s      46414 s     533699 s         20 s
#2  3500 MHz     367474 s       3338 s      41137 s     760140 s          0 s

  BLAS: libopenblas (USE64BITINT DYNAMIC_ARCH NO_AFFINITY Nehalem)
  LAPACK: libopenblas64_
  LIBM: libopenlibm
  LLVM: libLLVM-3.3
Environment:
  TERM = vt100
  LD_LIBRARY_PATH = :/usr/local/lib/
  PATH = /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/home/vagrant/julia/bin/
  JAVA_HOME = /usr/lib/jvm/java-7-oracle
  HOME = /home/vagrant

Package Directory: /home/vagrant/.julia/v0.4
2 required packages:
 - CauseMap                      0.0.3
 - JSON                          0.9.1
7 additional packages:
 - BinDeps                       0.4.7
 - Compat                        0.26.0
 - Conda                         0.5.3
 - MacroTools                    0.3.7
 - PyCall                        1.11.1
 - SHA                           0.3.3
 - URIParser                     0.1.8
INFO: Testing CauseMap

WARNING: deprecated syntax "{a=>b, ...}" at /home/vagrant/.julia/v0.4/CauseMap/src/coord_descent_tuning.jl:43.
Use "Dict{Any,Any}(a=>b, ...)" instead.

WARNING: deprecated syntax "[a=>b, ...]" at /home/vagrant/.julia/v0.4/CauseMap/src/coord_descent_tuning.jl:52.
Use "Dict(a=>b, ...)" instead.

WARNING: deprecated syntax "[a=>b, ...]" at /home/vagrant/.julia/v0.4/CauseMap/src/coord_descent_tuning.jl:54.
Use "Dict(a=>b, ...)" instead.
ERROR: LoadError: LoadError: LoadError: UndefVarError: Range1 not defined
 in include at ./boot.jl:261
 in include_from_node1 at ./loading.jl:333
 in require at ./loading.jl:273
 in include at ./boot.jl:261
 in include_from_node1 at ./loading.jl:333
 in include at ./boot.jl:261
 in include_from_node1 at ./loading.jl:333
 in process_options at ./client.jl:280
 in _start at ./client.jl:378
while loading /home/vagrant/.julia/v0.4/CauseMap/src/CauseMap.jl, in expression starting on line 107
while loading /home/vagrant/.julia/v0.4/CauseMap/test/testplotting.jl, in expression starting on line 1
while loading /home/vagrant/.julia/v0.4/CauseMap/test/runtests.jl, in expression starting on line 1
==============================[ ERROR: CauseMap ]===============================

failed process: Process(`/home/vagrant/julia/bin/julia --check-bounds=yes --code-coverage=none --color=no /home/vagrant/.julia/v0.4/CauseMap/test/runtests.jl`, ProcessExited(1)) [1]

================================================================================
ERROR: CauseMap had test errors
 in error at ./error.jl:21
 in test at pkg/entry.jl:803
 in anonymous at pkg/dir.jl:31
 in cd at file.jl:22
 in cd at pkg/dir.jl:31
 in test at pkg.jl:71
 in process_options at ./client.jl:257
 in _start at ./client.jl:378

>>> End of log
